Lab6.ipynb

```diff
@@ -0,0 +1,443 @@
+      "text/plain": [
+       "<Figure size 1296x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "r = stats.poisson.rvs(100,size=1000000)\n",
+    "plt.hist(r,bins=250)\n",
+    "plt.ylabel(\"Counts\", fontsize = '24')\n",
+    "plt.xlabel(\"Measurement\", fontsize = '24')\n",
+    "plt.title(\"Background generated by Poisson with mean 100\", fontsize = '30')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Problem 1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Find the 5-sigma sensitivity threshold."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The probability of a 5-sigma event for a standard normal distribution is 0.9999997133484281. The value associated to a 5-sigma event for a poisson distribution with a mean around 100 is 154.0. Thus the 5-sigma sensitivity threshold for our fake background is 154. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The threshold value is:  154.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "sig = stats.norm.cdf(5)   \n",
+    "ans= stats.poisson.ppf(sig,100)\n",
+    "print(\"The threshold value is: \", ans)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Problem 2a"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Inject a single signal multiple times into the background, between the range of 8-30 sigma."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We now need to generate the signal which will also be counting particle events. This means that our signal will also be Poisson distributed. We will pick 250 as the mean value for this Poisson signal injection, which falls within that range."
```

Comment:
Problem 2A:
- Explanation looks good and makes sense to me. Good clarifying that it refers to counting particle events in your explanation.
- Graph: Looks good. Typo in graph title: backgroung--> background. Axes labels readable and make sense.
- Did you use a large number of bins to show the discrete/Poisson nature of the distributions? (Just wondering). All looks good except minor typo.